inventory/admin.py

```diff
@@ -27,6 +30,41 @@ class OwnershipAdmin(admin.ModelAdmin):
 admin.site.register(Ownership, OwnershipAdmin)
 
 
+class OwnershipValueProxy(Ownership):
+    class Meta:
+        verbose_name = "Association inventory value"
+        proxy = True
+
+
+class OwnershipValueResource(ModelResource):
+    item_type = Field()
+    item_name = Field()
+    class Meta:
+        model = OwnershipValueProxy
+        fields = ('id', 'group__name', 'value')
+        export_order = ('id', 'group__name', 'item_type', 'item_name', 'value')
+
+    def dehydrate_item_type(self, ownership):
+        return str(ownership.content_object.__class__.__name__)
+
+    def dehydrate_item_name(self, ownership):
+        return str(ownership.content_object.name)
+
+
+@admin.register(OwnershipValueProxy)
+class OwnershipValues(ExportActionMixin, admin.ModelAdmin):
```

Comment:
As far as I can see, `OwnershipValueProxy` is only used to be able to create a second "ownership" management page in the admin panel, but with a limited queryset (being items owned by the association). Furthermore, I doubt we will ever want to export a subset of owned items, so the selection of items through an admin action seems unnecessary.

Instead, we could export via a [list filter](https://django-import-export.readthedocs.io/en/latest/getting_started.html#exporting-via-list-filters) and limit the queryset through `ExportMixin.get_export_queryset(..)` or even through the resource_class's `get_queryset(..)`. This way, the proxy model can be removed and all relevant information can be kept on a single admin page.

Reply:
What I tried to do, though I admit there are probably better ways than this proxy model hack, is make an additional Admin section that is focussed solely on the value of the inventory and not all other attributes (and additionally limited to only association owned items as only those values count). I think this might be a bit better for active content management, enabling the board to only access this list and not the complete list of options including contenttype and contentid.

Reply:
Even though there's still a lot of duplicate info, I suppose that's fair since the board will only be able to access the proxy model. I'm not entirely convinced with the current setup, but on the other hand the admin panel is quite finnicky to work with when wanting to do something non-standard. We might revisit it _at some point_ in the future if we encounter a better setup, but it'll definitely do for the time being.